budgetr-backend/src/test/java/pl/mantiscrab/budgetr/domain/bankaccount/BankAccountHelper.java

```diff
@@ -0,0 +1,41 @@
+package pl.mantiscrab.budgetr.domain.bankaccount;
+
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import pl.mantiscrab.budgetr.UriProvider;
+import pl.mantiscrab.budgetr.domain.bankaccount.dto.BankAccountDto;
+
+import java.net.URI;
+import java.util.List;
+
+import static org.springframework.web.servlet.mvc.method.annotation.MvcUriComponentsBuilder.on;
+
+public class BankAccountHelper {
```

Comment:
It's common that helper, utlity, service classes become god objects please be patient about that

Reply:
I thought about "BankAccountRequester" or "...""RequestHelper", but I thought simple "...Helper" will be better

Reply:
It's not about the name, but how you use this class